packages/token-balance-cache/package.json

```diff
@@ -12,13 +12,15 @@
   "keywords": [],
   "scripts": {
     "test": "ts-mocha --config ./.mocharc.json ./test/*.spec.ts",
-    "check-types": "tsc --noEmit",
```

Comment:
Any reason why this was removed?

Reply:
Lint is now handled at root. I think Timothee did this